docs/main-concepts/design-principles.md

```diff
@@ -160,13 +152,10 @@ library. Polyphony's design is based on the following principles:
   # use TCPServer from Ruby's stdlib
   server = TCPServer.open('127.0.0.1', 1234)
   while (client = server.accept)
-    spin do
+    spin {
       while (data = client.gets)
-        client.write('you said: ', data.chomp, "!\n")
+        client.write("you said: #{ data.chomp }\n")
       end
-    end
+    }
   end
   ```
-
-- Development of techniques and tools for converting callback-based APIs to
-  fiber-based ones.
```

Comment:
Combined this one with the bullet point that mentions the desire not to provide a callback API.